src/Couchbase.Lite.Shared/Sync/WebSocketWrapper.cs

```diff
@@ -295,6 +295,15 @@ private unsafe void Connected()
             });
         }
 
+        private void StopReachability()
+        {
+            if (_reachability != null)
```

Comment:
Also set to null here when done?

Reply:
This _reachability is readonly in WebSocketWrapper...

Reply:
Have you observed what happens if `StopReachability` is called twice somehow?